be/src/exec/parquet_reader.cpp

```diff
@@ -152,6 +166,43 @@ Status ParquetReaderWrap::init_parquet_reader(const std::vector<SlotDescriptor*>
     }
 }
 
+Status ParquetReaderWrap::get_schema(std::vector<std::string>* col_names, std::vector<TypeDescriptor>* col_types) {
```

Comment:
use SlotDescriptor can unify two params

Reply:
What we only need is the column name and the column type. There're some extra fields in SlotDescriptor.
https://github.com/StarRocks/starrocks/blob/e3864994919e18fa1eee5f21105c7e13fa2869c2/be/src/runtime/descriptors.h#L114-L127